akka-docs/src/main/paradox/typed/actor-discovery.md

```diff
@@ -14,30 +14,38 @@ To use Akka Actor Typed, you must add the following dependency in your project:
   version=$akka.version$
 }
 
-## Introduction
+## Obtaining Actor references
+
+There are two general ways to obtain @ref:[Actor references](../general/addressing.md#what-is-an-actor-reference): by
+@ref:[creating actors](#creating-actors) and by discovery using the @ref:[Receptionist](#receptionist).
 
 You can pass actor references between actors as constructor parameters or part of messages.
 
 Sometimes you need something to bootstrap the interaction, for example when actors are running on
 different nodes in the Cluster or when "dependency injection" with constructor parameters is not
 applicable.
 
+## Creating Actors
+
+An actor system is started by creating actors beneath the user guardian
+actor using the `ActorContext.spawn` method and then using
+`ActorContext.spawn` from within the created actors to spawn the actor
+tree. These methods return a reference to the newly created actor.
```

Comment:
maybe this text should be added to typed/actor-lifecycle.md instead, since that is already explaining how to create actors?

Reply:
Good idea

Reply:
In fact I think we can remove this all together, based on what is already in /typed/actor-lifecycle.md